pkg/controller/backup_controller.go

```diff
@@ -792,7 +792,7 @@ func (b *backupReconciler) runBackup(backup *pkgbackup.Request) error {
 		}
 	}
 
-	b.logger.WithField(Backup, kubeutil.NamespaceAndName(backup)).Info("Backup completed")
+	b.logger.WithField(Backup, kubeutil.NamespaceAndName(backup)).Info(fmt.Sprintf("Initial backup processing complete, moving to %s", backup.Status.Phase))
```

Comment:
Instead of `fmt.Sprintf`, can you use `InfoF`? (maybe aldo need to add line break `\n`)

Reply:
Infof is better than Sprintf into Info, although I don't think we generally include line breaks in log messages.

Reply:
Just worried that somethings add line break at the end (like fmt.Println), but others don't (like fmt.Printf), to do not have disorganized logs

Reply:
@mateusoliveira43 I'm not seeing any other log messages w/ `\n` in the codebase

Reply:
each log call should be single line. New line will always have timestamp in front.

Reply:
Thanks for the explanation, no line break needed :smile:

Reply:
Done, Updated the PR @mateusoliveira43 @sseago @kaovilai